task_01.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Provides a function to calculate party stats"""
+
+
+import data
+
+
+def get_party_stats(families, table_size=6):
+    """This function will calculate table to guest ratio.
+
+    Args:
```

Comment:
You should specify each argument in this section. Review the function documentation standards!